Tools/setup/ubuntu.sh

```diff
@@ -123,8 +123,8 @@ if [[ $INSTALL_NUTTX == "true" ]]; then
 		bison \
 		build-essential \
 		flex \
-		g++-multilib \
-		gcc-multilib \
+		g++-9-multilib-arm-linux-gnueabihf \
```

Comment:
Isn't there a `g++-multilib-arm-linux-gnueabihf`

(without the `9`)

Reply:
ok, I will use that one instead

Reply:
Did you check if it works?

Reply:
I only checked the one with 9. The one with 9 is absolutely working but I did not check the one without 9.
I need to use a clean jetson to test it.
I have two new jetsons on my desk.

Reply:
I verified it. It works. In fact `g++-multilib-arm-linux-gnueabihf` installs `g++-9-multilib-arm-linux-gnueabihf` itself. But it will be the latest version always.